backend/Shared/HitMeApp.Shared.Infrastructure/Exceptions/ExceptionResponse.cs

```diff
@@ -7,7 +7,7 @@ public class ExceptionResponse
         public object Response { get; }
         public HttpStatusCode StatusCode { get; }
 
-        public ExceptionResponse(object response, HttpStatusCode statusCode)
+        public ExceptionResponse(object response, HttpStatusCode statusCode = HttpStatusCode.BadRequest)
```

Comment:
Why are we defaulting it to `HttpStatusCode.BadRequest`?

Reply:
It's more frequent as we will usually encapsualte business logic errors which fits better with HTTP 400.
I resered HTTP 500 for unknown errors outside of module's scope. That would make exception mappers look cleaner.